src/pretix/control/templates/pretixcontrol/organizers/customer.html

```diff
@@ -48,6 +48,8 @@ <h3 class="panel-title">
                             </dd>
                             <dt>{% trans "Name" %}</dt>
                             <dd>{{ customer.name }}</dd>
+                            <dt>{% trans "Phone" %}</dt>
+                            <dd>{{ customer.phone }}</dd>
```

Comment:
This should be wrapped by an {% if %} like in the presale view, otherwise it will show a literal, untranslated "None"

Reply:
Ah, thanks – I tested it with an existing account and deleting the phone number just sets it blank, but initially creating an account sets it `None`. Fixed.